lib/client/layout.js

```diff
@@ -106,6 +106,24 @@ BlazeLayout._buildRegionGetter = function _buildRegionGetter(key) {
   };
 };
 
+BlazeLayout._getTemplate = function (layout, rootDomNode) {
+  if (Blaze._getTemplate) {
```

Comment:
Could you add the reason why we need this. May be link to this PR where you mention it with examples?